website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -74,9 +74,9 @@ The following arguments are supported:
     restart all of the instances at once. In the future, as the GCE API matures
     we will support `"ROLLING_UPDATE"` as well.
 
-* `target_size` - (Optional) If not given at creation time, this defaults to 1.
-    Do not specify this if you are managing the group with an autoscaler, as
-    this will cause fighting.
+* `target_size` - (Optional, Default `0`) The target number of running instances for this managed
```

Comment:
We use that style in a few places, I think I happened to have seen one of those right before writing this. Updated.